src/accelerate/utils/megatron_lm.py

```diff
@@ -1186,6 +1231,184 @@ def load_checkpoint(self, input_dir):
         if args.fp16 and self.iteration == 0:
             self.optimizer.reload_model_params()
 
+    def generate(
```

Comment:
Very reluctant to add this here with the same name as in Transformers. This method does not have the same signature/behavior so users will be very confused to get something different.

Reply:
renamed to `megatron_generate`